src/lib/encrypt.ts

```diff
@@ -1,43 +0,0 @@
-import { kmsClient } from "./utils/kms/kmsClient";
-import { changeId } from "./utils/tools/changeId";
-
-export default async function encrypt(
-  plaintextData: string,
-  keyName: string,
-): Promise<any> {
-  if (!plaintextData || !keyName || !process.env.kmsProjectId) {
-    console.log(
-      "Please specify both plaintextData/keyName/process.env.kmsProjectId",
-    );
-    throw new Error(
-      "Please specify both plaintextData/keyName/process.env.kmsProjectId",
-    );
-  }
-
-  const safeId = changeId(keyName);
-
-  const name = kmsClient.cryptoKeyPath(
-    process.env.kmsProjectId,
-    "global",
-    safeId,
-    "encryptDecrypt",
-  );
-
-  try {
-    const [encryptResponse] = await kmsClient.encrypt({
-      name,
-      plaintext: Buffer.from(plaintextData),
```

Comment:
This would actually throw an error if someone sends binary data to `encrypt` or `decrypt` (instead of sending a plain `string`), as it would be stringified using `JSON.stringify`, so `plaintextData = { 0: 23, 1: 65, 2: 67, 3: 23, ... }`.